analyses/cell-types-annotation/README.md

```diff
@@ -8,7 +8,8 @@ Parameters according to the project and analysis strategy will need to be specif
 - `project_parameters.Config.yaml` located at the `root_dir`
 - `future_globals_value` is hardwired coded in the `run-cell-types-annotation.R`. If necessary, user can increase/decrease resources.
 - Celldex references are hardwired coded in the `run-cell-types-annotation.R`. User should modify the reference to be used according to their experiment.
-
+- `run-cell-types-annotation.sh`: User should comment in/out the script to use based on thr selection of the desired methods for cell type annotation.
+- `04-merge-cell-types-annotations-all.R`: If more than one method was used for cell type annotation, user needs to run this script to merge the annotations. 
```

Comment:
I think this explanation could be clarified a bit more to indicate that even if running SingleR only, `04-merge-cell-types-annotations-all.R` should still be run if you want to merge `broad` and `fine` labeling results.